docs/antrea-network-policy.md

```diff
@@ -1766,6 +1784,20 @@ cluster-wide group.
   ignored, and the policy will have no effect.
   For a same ClusterGroup, `ipBlock` and `ipBlocks` cannot be set concurrently.
 
+  Starting from Antrea v2.2, Antrea-native policies now support an `except` field under
+  `ipBlocks`, which can be used to exclude specific CIDRs in the base IP range:
+
+  ```yaml
+  ipBlocks:
+  - cidr: 10.168.1.0/16
```

Comment:
`10.168.1.0/16` is strictly correct as a CIDR, it should be `10.168.0.0/16` in doc to avoid confusion, even though the code may be tolerant.

Reply:
Done